project.py

```diff
@@ -336,29 +342,56 @@ def area_code_prefixes() -> list:
     return output_list
 
 
+def show_summary_data(title, data):
+    data_table = {title: data}
+    df = pd.DataFrame(data_table)
+    print(f'=== {title} ===')
+    # If there are more than 10 values show only the top-10
+    print(df.sort_values(by=title, ascending=False).head(10))
+    print('Total', end=': ')
+    print(df[title].sum())
+
+    # make a bar chart of the data
+    fig = df.plot.pie(y=title, figsize=(10, 5), title=title, legend=False)

```

Comment:
For the social security number pie chart, the state abbreviations are too close to each other making it difficult to read at first. The solutions will be to increase the size of the figure, to add more colors to differentiate one state from another, and add a legend. You can create a variable named colors that contains a list of colors inside show_summary_data(title, data), and change line 355 into fig = df.plot.pie(y=title, figsize=(10, 5), title = title, legend = true, colors = colors);